cfme/tests/webui/test_advanced_search.py

```diff
@@ -1,127 +1,237 @@
+    else:
+        pytest.fail("Filter wasn't created or filters tree is not displayed!")
+
+
+def _select_filter(filters, filter_name, param):
+    if param.my_filters:
+        if isinstance(param.my_filters, tuple):
+            filters.tree.click_path(param.my_filters[1], "My Filters", filter_name)
+        else:
+            filters.tree.click_path("My Filters", filter_name)
+    else:
+        filters.navigation.select(filter_name)
+
+
+def test_advanced_search_button_displayed(param, appliance):
     """
     Polarion:
         assignee: anikifor
         casecomponent: web_ui
         caseimportance: medium
         initialEstimate: 1/30h
     """
-    view = _navigation(params, appliance)
+    view = _navigation(param, appliance)
     if not view.search.is_advanced_search_possible:
         pytest.fail(
             "Advanced search button is not displayed for {}_{}".format(
-                params[2], params[1].lower())
+                param.entity, param.destination.lower())
         )
 
 
-def test_can_open_advanced_search(params, appliance):
+def test_can_open_advanced_search(param, appliance):
     """
     Polarion:
         assignee: anikifor
         casecomponent: web_ui
         caseimportance: medium
         initialEstimate: 1/30h
     """
-    view = _navigation(params, appliance)
+    view = _navigation(param, appliance)
     view.search.open_advanced_search()
     if not view.search.is_advanced_search_opened:
         pytest.fail("Advanced search cannot be opened for {}_{}".format(
-            params[2], params[1].lower())
+            param.destination, param.entity.lower())
         )
+
+
+@pytest.mark.uncollectif(lambda param, appliance: (
+    (param.collection in (MyService, 'physical_providers', 'physical_servers', 'volume_backups',
+                          'volume_snapshots') and appliance.version < '5.9') or
+    (param.collection in (ConfigManager, 'ansible_tower_providers') and appliance.version > '5.10')
+    or (param.filter == 'Template (Ansible Tower) : Name' and appliance.version < '5.10') or
+    (param.filter == 'Job Template (Ansible Tower) : Name' and appliance.version > '5.10'))
+    or (param.collection == TemplatesImages and BZ(1626579)) or (param.collection == MyService
+                                                                 and BZ(1627078)))
+def test_filter_crud(param, appliance):
+    filter_name = fauxfactory.gen_string('alphanumeric', 10)
+    filter_value = fauxfactory.gen_string('alphanumeric', 10)
+    filter_value_updated = fauxfactory.gen_string('alphanumeric', 10)
+    view = _navigation(param, appliance)
+    # create
+    view.search.save_filter(
+        "fill_field({}, =, {})".format(param.filter, filter_value), filter_name)
+    view.search.close_advanced_search()
+    view.flash.assert_no_error()
+    # read
+    if param.my_filters:
+        if isinstance(param.my_filters, tuple):
+            filters = operator.attrgetter(param.my_filters[0])(view)
```

Comment:
**REQUIRED**
I think this is slightly more readable.
```python
filters = getattr(view, param.my_filters[0])
```

Reply:
@quarckster getattr doesn't work for example with 'sidebar.configured_systems'